cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,136 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should have the ability to navigate pages', () => {
+    cy.addWorker(100);
+    cy.get('select[aria-label="rows per page"] option').each(($option) => {
+      const optionText = $option.text();
+      cy.get('select[aria-label="rows per page"]').select(optionText);
+      cy.get('.-next button').should('exist').should('not.be.disabled');
+      cy.get('.-next button').click();
+      cy.get('.-pageJump input[type="number"]').should('have.value', '2');
+      cy.get('.-previous').click();
+      cy.get('.-pageJump input[type="number"]').should('have.value', '1');
+    });
+  });
+
+  it('should have the ability to selection count of rows', () => {
```

Comment:
```suggestion
  it('should have the ability to selection count of rows', () => {
```
add an assertion here, that table also contain selected amount of rows